pipelines/pipeline_navvis_rig.py

```diff
@@ -0,0 +1,305 @@
+import argparse
+import shutil
+import tempfile
+from pathlib import Path
+from typing import List, Optional
+
+from scantools import (
+    logger,
+    run_meshing,
+    run_navvis_to_capture,
+    run_rendering,
+    to_meshlab_visualization,
+)
+from scantools.capture import Capture
+from scantools.run_qrcode_detection import run_qrcode_detection_session
+from scantools.scanners.navvis.camera_tiles import TileFormat
+
+TILE_CHOICES = sorted([attr.name.split("_")[1] for attr in TileFormat])
+
+description = """
+--------------------------------------------------------------------------------
+
+Pipeline description
+====================
+
+Converts Navvis data to Lamar/Capture format exported as rig, including meshing,
+depth maps rendering, and QR code detection.
+
+**Input Parameters**
+
+    a) The input path for the data to be processed. Example:
+
+        datasets_proc/                           (--input_path "datasets_proc/")
+        ├── 2023-12-08_10.51.38
+        ├── 2023-12-15_15.45.51
+        └── 2023-12-15_15.58.10
+
+    b) The tile format. Default: "3x3".
```

Comment:
b) Large camera images can be tiled into smaller also overlapping images. The tile format specifies the tiling algorithm. The following values are accepted: ['3x3,'5x5'], where '3x3' is the default value.

Reply:
I feel this could be more descriptive